test/utils/test_PIDHandler.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+from edm4hep import ReconstructedParticleCollection, ParticleIDCollection, ReconstructedParticle
+import edm4hep
+import cppyy
+
+
+def create_recos():
+    coll = ReconstructedParticleCollection()
+    for i in range(1, 10):
+        reco = coll.create()
+        reco.setPDG(i)
+    return coll
+
+
+def create_particle_ids(recos, likelihood):
+    coll = ParticleIDCollection()
+    for reco in recos:
+        pid = coll.create()
+        pid.setLikelihood(likelihood)
+        pid.setParticle(reco)
+        pid.addToParameters(1.23)
+        pid.addToParameters(3.14)
+    return coll
+
+
+class TestPIDHandler(unittest.TestCase):
+    def test_pid_handler_basics(self):
+        reco_coll = create_recos()
+        pid_coll1 = create_particle_ids(reco_coll, 1.0)
+        pid_coll2 = create_particle_ids(reco_coll, 2.0)
+
+        handler = edm4hep.utils.PIDHandler()
+        handler.addColl(pid_coll1)
+        handler.addColl(pid_coll2)
+
+        reco = reco_coll[0]
+        pids = handler.getPIDs(reco)
+        self.assertEqual(len(pids), 2)
+        self.assertAlmostEqual(pids[0].getParticle(), reco, places=5)
+        self.assertAlmostEqual(pids[1].getParticle(), reco, places=5)
```

Comment:
```suggestion
        self.assertEqual(pids[0].getParticle(), reco,)
        self.assertEqual(pids[1].getParticle(), reco)
```

These should always be true (and they are not even floating point numbers but actual objects).